hooks/emitter/channel.go

```diff
@@ -157,7 +156,7 @@ func (h *Hook) extractFungibleTokenPacket(
 			} else {
 				packet["acknowledgement"] = common.JsDict{
 					"status": "failure",
-					"reason": evMap[types.EventTypeWriteAck+"."+types.AttributeKeyAck][0],
+					"reason": evMap[types.EventTypeWriteAck+"."+types.AttributeKeyAckHex][0],
```

Comment:
Need to confirm if it can be used interchangeably, or needs to decode hex first. 

AttributeKeyAckHex
AttributeKeyDataHex

Reply:
as checked, needs to decode it first. fixed.